autoload/moline/file.vim

```diff
@@ -61,7 +61,7 @@ let s:special_filetype= {
       \}
 
 function! moline#file#is_not_specialfile() abort
-  if strlen(&ft)==0 || (has_key(s:special_filetype,&filetype) && s:is_not_term())
+  if has_key(s:special_filetype,&filetype) && s:is_not_term()
```

Comment:
This may cause 'no ft' window i.e. terminal window to have more components as it's used as a condition.